code/headers/moving_platform.hpp

```diff
@@ -50,11 +61,6 @@ namespace r2d2::moving_platform {
         virtual void move(const int8_t &distance) = 0;
         virtual void move(const int8_t &x, const int8_t &y) = 0;
 
-    public:
-        moving_platform_c(base_comm_c &comm) : base_module_c(comm) {
```

Comment:
Why is this deleted? This is necessary for the canbus

Reply:
It's not deleted it's changed his place inside the moving_platform.hpp as first public method

Reply:
Ah ok, my bad